data/profile.d/rhel.conf

```diff
@@ -9,11 +9,6 @@ profile_id = rhel
 os_id = rhel
 
 [Installation System]
-# The detection is disabled since #1645686.
-# can_detect_unsupported_hardware = True
-# can_detect_support_removed = True
-
-# Show a warning if SMT is enabled.
 can_detect_enabled_smt = True
```

Comment:
I think you want to keep the last comment line?

Reply:
I don't. But I can add it back if it bothers you.